### PR TITLE
Add getAbstractActions translation

### PR DESCRIPTION
**Pull request by timgerstel**
This pull request implements the following feature:
- Add dispatcher actions to iFrame adapter

Related: https://github.com/zowe/sample-iframe-app/pull/22

Signed-off-by: Timothy Gerstel <tgerstel@rocketsoftware.com>

### CODE REVIEW

File: bootstrap/web/iframe-adapter.js [hunk 1]
```diff
@@ -50,12 +50,6 @@ let messageHandler = function(message) {
                     delete ZoweZLUX.iframe.contextMenuActions[key];
                 }
                 return;
-            case 'viewportEvents.spawnContextMenu':
-                if(data.contextMenuItemIndex !== undefined){
-                    ZoweZLUX.iframe.contextMenuActions[key][data.contextMenuItemIndex].action();
-                    delete ZoweZLUX.iframe.contextMenuActions[key];
-                }
-                return;
```
Comment (DivergentEuropeans) [inline bootstrap/web/iframe-adapter.js:58]: Was this intentionally removed? I think this was added to make the Zowe context menu work for Iframes

Comment (timgerstel) [reply bootstrap/web/iframe-adapter.js:58]: Its a duplicate

Comment (timgerstel) [reply bootstrap/web/iframe-adapter.js:58]: https://github.com/zowe/zlux-app-manager/pull/302/files#diff-083b8438c11bd78ce9a7db4a2dd96b4406bed2405ae89f8ae054bc02b4d05dc9R47

**Review by DivergentEuropeans — COMMENTED** [review]
Code looks good, awaiting test

**Review by rpenny125 — APPROVED** [review]
I only tested getAbstractActions, but that function behaved the way I expected.

There are still issues with the underlying API in that "getAbstractActions" does not return objects that can be passed into "invokeAction", but that would be for a separate pull request.

**Pull request merged**